src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -1491,62 +1491,13 @@ internal static object GetConverter(Type type, TypeTable backupTypeTable)
                 return typesXmlConverter;
```

Comment:
@iSazonov, your last commit had 2 failures in `PowerShell-CI-linux`
Language Primitive Tests.Convertion with 'TypeConverterAttribute' should be before convertion with .Net Core intrinsic type convertor
```powershell
Exception calling "ConvertTo" with "2" argument(s): "Cannot convert value "*Method*" to type "System.Management.Automation.PSMemberTypes". Error: "*Method* is not a valid value for PSMemberTypes.""
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Api/LanguagePrimitive.Tests.ps1: line 44
44:         $result = [System.Management.Automation.LanguagePrimitives]::ConvertTo('*Method*', [System.Management.Automation.PSMemberTypes])

```
DeserializedMethods.Deserialized objects shouldn't ever have any methods (unless they are primitive known types)
```powershell
Cannot bind parameter 'MemberType'. Cannot convert value "*Method*" to type "System.Management.Automation.PSMemberTypes". Error: "*Method* is not a valid value for PSMemberTypes."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Scripting/DeserializedMethods.Tests.ps1: line 19
19:         $d | Get-Member -MemberType *Method* Add | Should -BeNullOrEmpty

```